packages/web-twig/src/Resources/components/Button/stories/ButtonDisabled.twig

```diff
@@ -0,0 +1,30 @@
+<section class="docs-Section">
+  <h2 class="docs-Heading">Disabled</h2>
+
+  <div class="Grid Grid--cols-1 Grid--tablet--cols-2 Grid--desktop--cols-3">
+    {% for size in sizes %}
+    <div>
+      <h3 class="docs-Heading">Size {{size}}</h3>
+      {% for color in colors %}
+      <div class="{% if color is same as('inverted') %}docs-Box {% endif %}my-600">
+        <div>
+          <Button color={color} size={size} isDisabled>Disabled {{ color }}</Button>
+          <Button color={color} size={size} isDisabled>
+            <svg class="mr-400" width="24" height="24" aria-hidden="true">
```

Comment:
As @crishpeen already mentioned, wouldn't the `Icon` be better?

Reply:
Fixed.